src/components/security_manager/include/security_manager/security_manager_impl.h

```diff
@@ -158,7 +164,13 @@ class SecurityManagerImpl : public SecurityManager,
    * @brief Notifiers for listeners.
    * Allows to notify that certificate should be updated
    */
-  void NotifyOnCertififcateUpdateRequired();
```

Comment:
Since this is public, it will need to be deprecated instead of removed.

Reply:
@jacobkeeler it was not removed. There just was added OVERRIDE macro for safety

Reply:
@AKalinich-Luxoft It was deleted, note the difference in spelling. Unfortunately, fixing a typo means renaming the function, which is a breaking change. As long as we deprecated the function with the typo, we should be fine.

Reply:
@jacobkeeler sorry, I missed difference in spelling. Old function was restored and marked as deprecated. Seee ff52d04eb98aeb40d13dd8fd596a82ed9158e99f